ms-windows/mingw/build.sh

```diff
@@ -119,7 +119,7 @@ echo "::endgroup::"
 # export DISPLAY=:99
 
 echo "::group::compile QGIS"
-mingw$bits-make -C"$BUILDDIR" -j"$njobs" DESTDIR="${installroot}" install VERBOSE=1
+mingw$bits-make -C"$BUILDDIR" -j"$njobs" DESTDIR="${installroot}" install # VERBOSE=1
```

Comment:
Is it related to this PR?

Reply:
No, I filed a separate one for it: https://github.com/qgis/QGIS/pull/54924

I'll rebase and remove the commit